src/components/login-form.js

```diff
@@ -1,14 +1,118 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
+import axios from "axios";
 
-export default function LoginForm() {
-  return <Wrapper>안녕 나는 로그인 폼!</Wrapper>;
-}
+export default function LoginForm({ isLoggedIn, setLoginStatus, loginCheck }) {
+  const [form, setForm] = useState({ email: "", password: "" });
+
+  const handleFormChange = (e) => {
+    setForm({
+      ...form,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const initForm = () => {
+    setForm({ email: "", password: "" });
+  };
+  const validCheck = () => {
+    if (form.email.length === 0 || form.password.length === 0) {
+      alert("모든 항목을 입력하세요");
+      return false;
+    }
+    return true;
+  };
+  const handleSubmit = () => {
+    console.log(form);
+    if (!validCheck) return;
+    axios
+      .post(process.env.API_HOST + "/auth/signin/", form)
+      .then(function (response) {
+        console.log(response);
+        alert("로그인 성공!");
+        setLoginStatus(true);
+        console.log(isLoggedIn);
+      })
+      .catch(function (error) {
+        initForm();
+        alert("로그인 실패!");
+        console.log(error);
+      });
+  };
 
+  return (
+    <Wrapper>
+      <LoginLabel>로그인</LoginLabel>
+      <InfoInputArea>
+        <EmailInputArea>
+          <EmailLabel>EMAIL</EmailLabel>
+          <EmailInput
+            name="email"
+            value={form.email}
+            onChange={handleFormChange}
+          />
+        </EmailInputArea>
+        <PasswordInputArea>
+          <PasswordLabel>PASSWORD</PasswordLabel>
+          <PasswordInput
+            name="password"
+            value={form.password}
+            onChange={handleFormChange}
+            type="password"
+          />
+        </PasswordInputArea>
+      </InfoInputArea>
+      <Submit onClick={handleSubmit}>로그인</Submit>
+    </Wrapper>
+  );
+}
+export const MemoizedLoginForm = React.memo(LoginForm);
 const Wrapper = styled.div`
   width: 100%;
   min-height: 30rem;
   background-color: white;
   font-size: 18px;
   padding: 3rem 4rem;
+  display: flex;
+  flex-direction: column;
+`;
+const LoginLabel = styled.p`
```

Comment:
```suggestion
const LoginLabel = styled.label`
```
input의 label이기 때문에 p 태그 대신 label 태그를 써도 좋겠네용